src/pages/projects/AddProjectContributorForm.tsx

```diff
@@ -81,7 +92,7 @@ export const AddProjectContributorForm = ({
         searchTerm={searchTerm}
         setSearchTerm={setSearchTerm}
       />
-      <StyledRightAlignedFooter sx={{ mt: '2rem' }}>
+      <StyledRightAlignedFooter sx={{ mt: '2rem', flexWrap: 'nowrap' }}>
```

Comment:
👀 
![bilde](https://github.com/user-attachments/assets/bdf81cf5-9289-4431-8966-84c14b9963a4)
```suggestion
      <StyledRightAlignedFooter sx={{ mt: '2rem' }}>
```